tasks/test_e2e.yml

```diff
@@ -0,0 +1,60 @@
+- name: Querying metrics in Prometheus
+  hosts: undercloud-0
+  become: true
+  become_method: sudo
+  become_user: root
+  tasks:
+    - name: "Extract URL from stf-connectors template"
+      shell: grep interconnect /home/stack/virt/stf-connectors-osp16-edge.yaml | sed 's;.*\(-service-telemetry.*\);prometheus-operated\1;'
+      register: prom_url
+      failed_when: "prom_url.stdout_lines|length == 0"
+    - debug:
+        var: prom_url.stdout_lines
+    
+    - debug: 
+        msg: The URL is {{ prom_url.stdout_lines }}
+
+    - name: "Capture ceph_cpu_bytes metrics"
+      shell: /usr/bin/curl -k http://{{ prom_url.stdout_lines[0] }}/api/v1/query? --data-urlencode 'query=collectd_ceph_ceph_bytes {plugin_instance="ceph-osd.0"}[1m]' > /tmp/query_ceph_cpu_bytes
+      register: checkmyconf
+      failed_when:
+        - checkmyconf.rc !=0 
+    
+    - name: "Capture collectd_cpu_percent"
+      shell: /usr/bin/curl -k http://{{ prom_url.stdout_lines[0] }}/api/v1/query? --data-urlencode 'query=collectd_cpu_percent {plugin_instance="0",type_instance="user",service="default-cloud1-coll-meter-smartgateway"}[1m]' > /tmp/query_collectd_cpu_percent
+      register: checkmyconf
+      failed_when:
+        - checkmyconf.rc !=0
+    
+    - name: "Capture collectd_iinterface_tx_total"
+      shell: /usr/bin/curl -k http://{{ prom_url.stdout_lines[0] }}/api/v1/query? --data-urlencode 'query=collectd_interface_if_packets_tx_total {type_instance="base"}[1m]' > /tmp/query_collectd_interface_tx_total
+      register: checkmyconf
+      failed_when:
+        - checkmyconf.rc !=0
+
+    - name: "[Test] Check whether ceph cpu  metrics received in prometheus for all ceph nodes"
+      command: egrep 'ceph' /tmp/query_ceph_cpu_bytes
+      register: checkmyconf
+      changed_when: no
+      failed_when:
+        - checkmyconf.rc != 0
+    - debug:
+        msg="COLLECTD CPU CEPH METRICS RECEIVED FOR ALL CEPH NODES"
+
+    - name: "[Test] Check whether collectd interface TX total  metrics received  in prometheus for all overcloud nodes"
```

Comment:
Same all vs. any problem here, I think.

Reply:
Since checking any is okay for now, I suggest updating the task name to reflect ANY vs all.

Reply:
> Since checking any is okay for now, I suggest updating the task name to reflect ANY vs all.

Done.